capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -67,13 +69,38 @@ export class SearchUserModal extends Component {
     this.setState({
       searchTerm: event.target.value,
       typingTimeout: setTimeout(async () => {
-        await this.getBooks(this.state.searchTerm)
+        await this.getUsers(this.state.searchTerm)
       }, 500)
     })
   }
 
   componentDidMount() {
     this.fetchCollaborators();
+
+    // TODO: Update ClubPage based on user input
+
+    if (this.props.club && this.props.type === 'clubs') {
+      if (this.props.club.memberIDs) {
+        if (this.props.club.memberIDs.length > 0) {
+          this.fetchMembers();
+        }
+      }
+    }
+  }
+
+  removeDuplicatesArrayJsonId = (array) => {
+
+    let helperArrayIDs = [];
+    let newArray = [];
```

Comment:
nit: `const` since these aren't being reassigned, you can still mutate them with `push`

Reply:
Ok